spec/spec_helper.rb

```diff
@@ -0,0 +1,4 @@
+require 'chefspec'
+require 'chefspec/berkshelf'
+
+at_exit { ChefSpec::Coverage.report! }
```

Comment:
I don't understand what this does. I know that tests work just fine without this.